autopilot/autopilot-serverless-inference/autopilot-models-serverless-inference.ipynb

```diff
@@ -0,0 +1,1311 @@
+{
```

Comment:
<p>Use present tense - (i.e. we, not we'll) - do this for the entire article</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>